src/pages/ws/ui/control-panel/zone-modals/tags-modal.tsx

```diff
@@ -15,7 +14,7 @@ export const TagsModal = observer(({ close, title }: ITagsModalProps) => {
   const [searchValue, setSearchValue] = useState('')
 
   useEffect(() => {
-    zoneStore.fetchZoneTagsAsync()
+    zoneStore.tags.length <= 0 && zoneStore.fetchZoneTagsAsync()
```

Comment:
i think length can't be less than 0. Also, please, use if() { } statement for this